__tests__/app.test.js

```diff
@@ -0,0 +1,41 @@
+const request = require('supertest');
+const app = require('../app');
+const db = require('../db/connection.js');
+
```

Comment:
we also want to be seeding the database before each test (see the notes on seeding with pg for a refresher!)